ocpp/v20/enums.py

```diff
@@ -0,0 +1,697 @@
+class _15118EVCertificateStatus:
+    accepted = "Accepted"
+    failed = "Failed"
+
+
+class APNAuthentication:
+    chap = "CHAP"
+    none = "NONE"
+    pap = "PAP"
+    auto = "AUTO"
+
+
+class Attribute:
+    actual = "Actual"
+    target = "Target"
+    min_set = "MinSet"
+    max_set = "MaxSet"
+
+
+class AuthorizationStatus:
+    accepted = "Accepted"
+    blocked = "Blocked"
+    concurrent_tx = "ConcurrentTx"
+    expired = "Expired"
+    invalid = "Invalid"
+    no_credit = "NoCredit"
+    not_allowed_type_evse = "NotAllowedTypeEVSE"
+    not_at_this_location = "NotAtThisLocation"
+    not_at_this_time = "NotAtThisTime"
+    unknown = "Unknown"
+
+
+class BootReason:
+    application_reset = "ApplicationReset"
+    firmware_update = "FirmwareUpdate"
+    local_reset = "LocalReset"
+    power_up = "PowerUp"
+    remote_reset = "RemoteReset"
+    scheduled_reset = "ScheduledReset"
+    triggered = "Triggered"
+    unknown = "Unknown"
+    watchdog = "Watchdog"
+
+
+class CertificateStatus:
+    accepted = "Accepted"
+    signature_error = "SignatureError"
+    certificate_expired = "CertificateExpired"
+    certificate_revoked = "CertificateRevoked"
+    no_certificate_available = "NoCertificateAvailable"
+    cert_chain_error = "CertChainError"
+    contract_cancelled = "ContractCancelled"
+
+
+class ChargingLimitSource:
+    ems = "EMS"
+    other = "Other"
+    so = "SO"
+    cso = "CSO"
+
+
+class ChargingProfileKind:
+    absolute = "Absolute"
+    recurring = "Recurring"
+    relative = "Relative"
+
+
+class ChargingProfilePurpose:
+    charging_station_external_constraints =\
+        "ChargingStationExternalConstraints"
+    charging_station_max_profile = "ChargingStationMaxProfile"
+    tx_default_profile = "TxDefaultProfile"
+    tx_profile = "TxProfile"
+
+
+class ChargingRateUnit:
+    w = "W"
+    a = "A"
+
+
+class ChargingState:
+    charging = "Charging"
+    ev_detected = "EVDetected"
+    suspended_ev = "SuspendedEV"
+    suspended_evse = "SuspendedEVSE"
+
+
+class ClearMonitoringStatus:
+    accepted = "Accepted"
+    rejected = "Rejected"
+    not_found = "NotFound"
+
+
+class ComponentCriterion:
+    active = "Active"
+    available = "Available"
+    enabled = "Enabled"
+    problem = "Problem"
+
+
+class Connector:
+    c_ccs1 = "cCCS1"
+    c_ccs2 = "cCCS2"
+    c_g105 = "cG105"
+    c_tesla = "cTesla"
+    c_type1 = "cType1"
+    c_type2 = "cType2"
+    s309_1p_16a = "s309-1P-16A"
+    s309_1p_32a = "s309-1P-32A"
+    s309_3p_16a = "s309-3P-16A"
+    s309_3p_32a = "s309-3P-32A"
+    s_bs1361 = "sBS1361"
+    s_cee_7_7 = "sCEE-7-7"
+    s_type_2 = "sType2"
```

Comment:
to be consistent with the previous nomenclature, it should be s_type2 and s_type3